package.json

```diff
@@ -5,13 +5,19 @@
   "description": "Foreman isn't really a node module, these are just dependencies needed to build the webpack bundle. 'dependencies' are the asset libraries in use and 'devDependencies' are used for the build process.",
   "private": true,
   "devDependencies": {
+    "babel-cli": "^6.10.1",
     "babel-core": "~6.7.2",
     "babel-eslint": "^6.1.2",
     "babel-loader": "~6.2.4",
+    "babel-plugin-transform-class-properties": "^6.11.5",
     "babel-preset-es2015": "~6.6.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
```

Comment:
Do we actually use any stage-0 features? I would recommend against using anything that isn't accepted as a standard already. (see https://babeljs.io/docs/plugins/#stage-x-experimental-presets for what the stages mean and the ECMAScript technical committee process)